pennylane/transforms/hamiltonian_expand.py

```diff
@@ -86,13 +81,36 @@ def hamiltonian_expand(tape, group=True):
             qml.PauliX(wires=2)
             qml.expval(H)
 
-        # split H into observable groups [qml.PauliZ(0)] and [qml.PauliX(1), qml.PauliX(0)]
-        tapes, fn = qml.transforms.hamiltonian_expand(tape)
-        print(len(tapes)) # 2
+    With grouping, the Hamiltonian gets split into two observable groups (here ``[qml.PauliZ(0)]`` and
+    ``[qml.PauliX(1), qml.PauliX(0)]``):
```

Comment:
```suggestion
    ``[qml.PauliX(1), qml.PauliX(0)]``):

```
This code block didn't render properly in the docs;  I think it's because there is no blank line before the line with the `>>>`